nnstreamer/tensor_trainer/tensor_trainer_nntrainer.cc

```diff
@@ -109,14 +111,14 @@ void init_subplugin_nntrainer(void) __attribute__((constructor));
  */
 void fini_subplugin_nntrainer(void) __attribute__((destructor));
 
-void nntrainer_thread_func(NNTrainer::NNTrainerTrain *nntrainer) {
-  nntrainer->trainModel();
+void nntrainer_thread_func(SubPlugin::Trainer *trainer) {
```

Comment:
is there any reason to change the name??

Reply:
I think nntrainer uses 'NNTrainer' and 'nntrainer' keywords as very important keywords.
but,
When we grep from the code to 'NNTrainer' and 'nntrainer', a lot of sub-plugin code comes out, so I changed the name because I thought it would become an unnecessary list for developers when developing nntrainer core.